server/go.mod

```
@@ -223,7 +226,11 @@ require (
 	modernc.org/sqlite v1.39.1 // indirect
 )
 
-// Also prevent tablewriter from being upgraded because the downstream dependency
-// jaytaylor/html2text does not have a go.mod file which makes it bump to the latest
-// version always. Tablewriter has made breaking changes to its latest release.
+// Prevent tablewriter from being upgraded because the downstream dependency 
+// code.sajari.com/docconv/v2 has am indirect dependency on jaytaylor/html2text via 
+// advancedlogic/GoOse. jaytaylor/html2text does not have a go.mod file which makes 
+// it bump to the latest version always. Tablewriter has made breaking changes to its 
+// latest release.
+// There is a proposed PR to fix this for GoOse we should monitor: 
+// https://github.com/advancedlogic/GoOse/pull/77
```

Thank you for the detailed comment! There's a small typo:
```suggestion
// Prevent tablewriter from being upgraded because the downstream dependency 
// code.sajari.com/docconv/v2 has an indirect dependency on jaytaylor/html2text via 
// advancedlogic/GoOse. jaytaylor/html2text does not have a go.mod file which makes 
// it bump to the latest version always. Tablewriter has made breaking changes to its 
// latest release.
// There is a proposed PR to fix this for GoOse we should monitor: 
// https://github.com/advancedlogic/GoOse/pull/77
```